collector/settings.mdx

```diff
@@ -174,6 +174,14 @@ helper functions are expected to be defined, and the one we connect to for serve
       <td>[none]</td>
       <td>Alternative to above, using actual contents of the key</td>
     </tr>
+    <tr>
+      <td><code>DB_ALL_NAMES</code></td>
+      <td>false</td>
+      <td>Alternative to setting <code>*</code> at the last entry of db_name (<code>DB_NAME</code>),
+      monitoring all databases on server. Can be only set using an environment variable.<br />
```

Comment:
+1 on the updated wording, and a few more suggestions:

```suggestion
      <td>Alternative to setting <code>*</code> as the last entry of db_name (<code>DB_NAME</code>) to
      monitor all databases on the server. Can only be set using an environment variable.<br />
```

I think we left this undocumented intentionally because the wildcard syntax came later, but the use case you describe below seems reasonable, so no objection to adding this to the docs.